docs/guides/private-import.md

```diff
@@ -2,47 +2,76 @@
 tags:
   - 👷 Staff
 ---
-!!! warning "This tutorial might be partial or incomplete. Please check with the team if you have questions."
-
-Most of the steps have private versions with `-private` suffix (e.g. `data-private://...`, `walden-private://...`) that remember and enforce a dataset level `is_public` flag.
-
-When publishing, the index is public, but tables in the index that are private are only available over s3 with appropriate credentials. `owid-catalog-py` is also updated to support these private datasets and supports fetching them over s3 transparently.
-
-### Uploading private data to walden
-
-It is possible to upload private data to walden so that only those with S3 credentials would be able to download it.
-
-```python
-from owid.walden import Dataset
-
-local_file = 'private_test.csv'
-metadata = {
-  'name': 'private_test',
-  'short_name': 'private_test',
-  'description': 'testing private walden data',
-  'source_name': 'test',
-  'url': 'test',
-  'license_url': 'test',
-  'date_accessed': '2022-02-07',
-  'file_extension': 'csv',
-  'namespace': 'private',
-  'publication_year': 2021,
-}
-
-# upload the local file to Walden's cache
-dataset = Dataset.copy_and_create(local_file, metadata)
-# upload it as private file to S3
-url = dataset.upload(public=False)
-# update PUBLIC walden index with metadata
-dataset.save()
+
+While most of the data at OWID is publicly available, some datasets are added to our catalogue with some restrictions. These include datasets that are not redistributable, or that are not meant to be shared with the public. This can happen due to a strict license by the data provider, or because the data is still in a draft stage and not ready for public consumption.
+
+Various privacy configurations are available:
+
+- Skip re-publishing to GitHub.
+- Disable data downloading options on Grapher.
+- Disable public access to the original file (snapshot).
+- Hide the dataset from our public catalog (accessible vua `owid-catalog-py`).
+
+In the following, we explain how to create private steps in the ETL pipeline and how to run them.
+
+## Create a private step
+
+### Snapshot
+
+To create a private snapshot step, set the `meta.is_public` property in the snapshot .dvc file to false:
+
+```yaml
+meta:
+  is_public: false
+
+  origin:
+    # Data product / Snapshot
+    title: World Population Prospects
+    ...
+```
+
+This will prevent the file to be publicly accessible without the appropriate credentials.
+
+### Meadow, Garden, Grapher
+
+Creating a private data step means that the data is not listed in the public catalog. Also, private datasets will not be re-published to GitHub.
```

Comment:
Maybe it's good to clarify what "the public catalog" means. And what "republished to Github" means.